### ISSUE TITLE
client-all-2-11 doesn't include sources

### ISSUE THREAD

**Issue by sschaef** [sbt/sbt-remote-control#282]
The sources jar contains only a manifest, but beside from that it is empty:
```
% jar tf ~/.m2/repository/com/typesafe/sbt/client-all-2-11/0.3.2/client-all-2-11-0.3.2-sources.jar
META-INF/MANIFEST.MF
```

**Comment by jsuereth** [comment]
This would be a great fix if you want to take a crack at it.

CLient-all is an aggregate/assembly jar.  You'd need to also aggregate/assemble the sources.


Example code:

Where we borrowed the code from:
https://github.com/typesafehub/sbt-republish/blob/master/project/SbtRepublish.scala#L150-L166

Where it is now:
https://github.com/sbt/sbt-remote-control/blob/master/project/sbtrc.scala#L60-L80


Basically, we should create a "RepackageSources" configuration and then mirror the assembly settings in this configuration and feed all the source jars to it (see: https://github.com/typesafehub/sbt-republish/blob/master/project/SbtRepublish.scala#L130).

If I have time I'll try to knock it out, otherwise feel free to ask questions if you're interested.
